analyzer/src/funTest/kotlin/GitRepoTest.kt

```diff
@@ -67,7 +67,7 @@ class GitRepoTest : StringSpec() {
             enabled = !Ci.isAzureWindows && !Ci.isTravis
         ) {
             val ortResult = Analyzer(DEFAULT_ANALYZER_CONFIGURATION).analyze(outputDir)
-            val actualResult = yamlMapper.writeValueAsString(ortResult)
+            val actualResult = ortResult.toYaml()
```

Comment:
Just my 2ct: I'm a bit undecided whether having this helper function really is beneficial. The amount of code is not really significantly reduced, and neither is the readability significantly increased IMO.

Reply:
Well, I like it. Btw, a nice side effect is that it removes the false positive IntelliJ warning about inappropriate use of a blocking method call.

Reply:
Ah, removing that false positive now is something that convinces me, as it has annoyed me in the past, too. Please add that to the commit message as I wouldn't have complained then in the first place :smile:

Reply:
Done.